bench-vortex/src/tpch/dbgen.rs

```diff
@@ -58,6 +58,14 @@ impl DBGenOptions {
             cache_dir: self.cache_dir,
         }
     }
+
+    pub fn with_scale_factor(self, scale_factor: u8) -> Self {
```

Comment:
We should make an issue to followup by

- Killing our dbgen wrapper
- Replacing it with duckdb, which can [generate TPC-H](https://duckdb.org/docs/extensions/tpch.html) data and [upload it to an S3 bucket](https://duckdb.org/docs/guides/network_cloud_storage/s3_export.html)